src/Eventso.Subscription.Kafka.DeadLetter.Postgres/DistributedMonitor.cs

```diff
@@ -0,0 +1,33 @@
+using System.Threading;
+using System.Threading.Tasks;
+using Npgsql;
+
+namespace Eventso.Subscription.Kafka.DeadLetter.Postgres
+{
+    public static class DistributedMonitor
+    {
+        public static async Task<bool> TryEnter(NpgsqlConnection connection, long lockId, CancellationToken token)
```

Comment:
it can be extension method for connection. And I think this class should be internal

Reply:
will discuss it later